dask/dataframe/tests/test_groupby.py

```diff
@@ -2475,3 +2475,26 @@ def agg(grp, **kwargs):
         agg(pdf.groupby(groupby, observed=observed)),
         agg(ddf.groupby(groupby, observed=observed)),
     )
+
+
+def test_empty_partitions_with_value_counts():
+    # https://github.com/dask/dask/issues/7065
+    df = pd.DataFrame(
+        data=[
+            ["a1", "b1"],
+            ["a1", None],
+            ["a1", "b1"],
+            [None, None],
+            [None, None],
+            [None, None],
+            ["a3", "b3"],
+            ["a3", "b3"],
+            ["a5", "b5"],
+        ],
+        columns=["A", "B"],
+    )
+
+    expected = df.groupby("A")["B"].value_counts()
+    ddf = dd.from_pandas(df, npartitions=3)
+    actual = ddf.groupby("A")["B"].value_counts().compute()
+    assert_eq(expected, actual)
```

Comment:
`assert_eq` will call `compute` internally and will also perform some additional sanity checks when a Dask DataFrame is used as an input

```suggestion
    actual = ddf.groupby("A")["B"].value_counts()
    assert_eq(expected, actual)
```

Reply:
Thanks! That's what I get for just copying the original MRE :)